configure.ac

```diff
@@ -102,6 +102,13 @@ AC_ARG_WITH([stage1-systemd-version],
             [RKT_STAGE1_SYSTEMD_VER="${withval}"],
             [RKT_STAGE1_SYSTEMD_VER='auto'])
 
+# STAGE1 - kvm hypervisor
+AC_ARG_WITH([stage1-kvm-hypervisor],
+            [AS_HELP_STRING([--with-stage1-kvm-hypervisor],
+                            [hypervisor used by 'kvm' flavor; choose from: 'lkvm', default: 'lkvm'])],
+            [RKT_STAGE1_KVM_HV="${withval}"],
+            [RKT_STAGE1_KVM_HV='qemu'])
```

Comment:
Setting up `qemu` as a default value is temporary - by default it should be `lkvm`